components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2870,4 +2870,22 @@ void RewardsServiceImpl::OnDeleteActivityInfo(
   }
 }
 
+void RewardsServiceImpl::RefreshPublisher(
+    RefreshPublisherCallback callback,
+    const std::string& publisher_key) {
+  if (!Connected())
+    return;
```

Comment:
we need to call back here with false and empty string

Reply:
Fixed